dev-docs/bidders/exadsadserver.md

```diff
@@ -0,0 +1,487 @@
+---
+layout: bidder
+title: EXADS
+description: EXADS Bidder Adapter
+pbjs: true
+pbs: false ?
+biddercode: exadsadserver
+gvl_id: 1084 ? // found here https://iabeurope.eu/vendor-list-tcf/
+tcfeu_supported: true?
+usp_supported: false?
+media_types: banner, video, native
+gpp_sids: ?
+schain_supported: ?
+dchain_supported: ?
+safeframes_ok: ?
+deals_supported: ?
+floors_supported: ?
+ortb_blocking_supported: ?
+---
+
+### Note
+
+Module that connects to EXADS’ bidder for bids.
+
+### Configuration
+
+Use `setConfig` to instruct Prebid.js to initilize the exadsBidAdapter, as specified below. 
+* Set "debug" as true if you need to read logs;
+* Set "gdprApplies" as true if you need to pass gdpr consent string;
+* The tcString is the iabtcf consent string for gdpr;
+* Uncomment the cache instruction if you need to configure a cache server (e.g. for instream video)
+
+```
+pbjs.setConfig({
+    debug: false,
+    //cache: { url: "https://prebid.adnxs.com/pbc/v1/cache" },
+    consentManagement: {
+        gdpr: {
+            cmpApi: 'static',
+            timeout: 1000000,
+            defaultGdprScope: true,
+            consentData: {
+                getTCData: {
+                    tcString: consentString,
+                    gdprApplies: false // set to true to pass the gdpr consent string
+                }
+            }
+        }
+    }
+});
+```
+Add the `video` config if you need to render videos using the video module.
+For more info navigate to https://docs.prebid.org/prebid-video/video-module.html.
+```
+pbjs.setConfig({
+    video: {
+        providers: [{
+            divId: 'player', // the id related to the videojs tag in your body
+            vendorCode: 2, // videojs, 
+            playerConfig: {
+                params: {
+                    adPluginConfig: {
+                        numRedirects: 10
+                    },
+                    vendorConfig: {
+                        controls: true,
+                        autoplay: true,
+                        preload: "auto",
+                    }
+                }
+            }
+        },]
+    },
+});
+```
+
+#### RTB Banner 2.4
+
+{: .table .table-bordered .table-striped }
+| Name          | Scope    | Description           | Example   | Type      |
+|---------------|----------|-----------------------|-----------|-----------|
+| `zoneID`      | required | You can get it from the endpoint created after configuring the zones          | `12345`    | `'integer'` |
```

Comment:
@giuseppe-exads Having both " \` " and " ' " for the type is too much. Use " \` " only.

Reply:
done